nav2_costmap_2d/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 project(nav2_costmap_2d)
-
+#temp--------------------------------------
+set(FastRTPS_INCLUDE_DIR /opt/ros/foxy/include)
```

Comment:
This is anything relating to fast-rtps required for this work?

Reply:
ROS installation of Debian package would cause the problem that FastRTPS path lost, while source code installation doesn't have this problem, it will be removed, cause it's not related with this work.

Reply:
I'm not sure I understand the comment, but a-ok